### ISSUE TITLE
Firebase and Hasura Not Syncing

### ISSUE THREAD

**Issue by studuino** [FarazPatankar/firebase-hasura-sync#1]
Hi Sir,
I have an issue with syncing firebase and hasura. 
Users are registered in firebase as expected.
But the cloud function seems not sync to my hasura graphql endpoint.

Errors shown as folllows in:
(1) react-expo browser 
![image](https://user-images.githubusercontent.com/2291009/82039964-c83a1a80-96cb-11ea-96d2-68b3ab173414.png)

(2) Firebase 
![image](https://user-images.githubusercontent.com/2291009/82040092-07686b80-96cc-11ea-94d3-f45d86842fa6.png)

Tried with insert mutation(using role : admin) from postman with success
![image](https://user-images.githubusercontent.com/2291009/82040343-7b0a7880-96cc-11ea-860c-4cbfe61df1da.png)

Thanks.

**Comment by FarazPatankar** [comment]
Have you upgraded Firebase to the Blaze plan? Hasura is an external service and Firebase requires that you upgrade to the Blaze plan in order to make requests to external services.

If you already have, it might make sense to try logging the errors in your cloud function and then deploying again as I can't help much unless I know what the error is.

**Comment by studuino** [comment]
Hi @FarazPatankar 
Thank you for your advice. Mine is Blaze plan.
Now I deployed hasura in heroku and got secure hasura endpoint url (https).
Previously I deployed in docker without having https.
I am not so sure it is relevant to my problem or not.

Now I got different error as follows:
![image](https://user-images.githubusercontent.com/2291009/82115286-cc2b7280-9787-11ea-9290-ae52d8e69e26.png)

Thanks.
